playground-python.mdx

```diff
@@ -0,0 +1,64 @@
+
+To achieve this, we use rollup to bundle the code and a [plugin](https://github.com/SferaDev/rollup-plugin-import-cdn) to load any external library from a CDN. This way, we can transpile the code and all its dependencies into a single file that can be executed in a separate thread using [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers).
+
+Running the code in a separate thread is important, as it allows us to avoid blocking the main thread. This way, the UI is always responsive, and you can continue writing code while the previous code is being executed. Also, it provides some built-in security, as the code is isolated from the main thread.
+
+Also, to improve the experience when writing TypeScript code, we switched to the fork of monaco-editor that [TypeScript Playground](https://www.typescriptlang.org/play) uses. This fork is always up to date with the latest version of TypeScript, and it also includes several improvements for TypeScript developers.
+
+## Adding support for multiple files and languages
+
+The first version of the Playground only supported a single TypeScript file. This was a limitation that we wanted to remove, and when we started working on it, we also decided to add support for other languages.
+
+Monaco is the open source editor that powers VS Code, and it has support for multiple files and languages. We just needed to use its virtual file system that would allow us to load multiple files and execute them separately.
+
+The refactor allowed us to add support for SQL, both for full file execution and for inline execution. This was a great addition, as it allowed us to show how Xata's SQL Proxy works, and how you can use it to query your data.
+
+## Adding support for Python
+
+The next step was to add support for Python. We wanted to have the same experience as with TypeScript and SQL, where you could write code and execute it in the browser. Luckily, there are several projects that allow you to run Python in the browser, and we decided to use [Pyodide](https://pyodide.org).
+
+Pyodide is a project started by Mozilla that allows you to run Python in the browser using WebAssembly. It includes the standard library and several popular packages like NumPy, Pandas, or Matplotlib. It also includes a package manager that allows you to install any other package with a wheel available on PyPI.
+
+Similarly to TypeScript, we have a web worker that executes the Python code. The code is executed by Pyodide, and we load external packages from imports using the package manager.
+
+To make it fully work with our Python SDK, we only needed patch the runtime so that Pyodide could do HTTP requests, using `pyodide-http`.
+
+## Conclusion
+
```

Comment:
Maybe another picture with a complex python program now?

Reply:
Yeah, I was going to suggest the same. Right after we introduce Python, let's show the new UI with a few saved queries and Python support